core/org.eclipse.cdt.core/src/org/eclipse/cdt/core/build/ICBuildConfiguration.java

```diff
@@ -79,9 +80,21 @@ default String getLaunchMode() {
 	 *
 	 * @return binary parser ids
 	 * @throws CoreException
+	 * @deprecated As of 8.0 replaced by {@link ICBuildConfiguration#getBinaryParserIds}
 	 */
+	@Deprecated(since = "8.0")
```

Comment:
Delete this, don't just deprecate it.

Reply:
The reason to bump the major version for this is precisely so we can delete the API without deprecation.
https://github.com/eclipse-cdt/cdt/blob/main/doc/org.eclipse.cdt.doc.isv/guide/deprecated_API_removals.html needs to be updated to list the deprecation + mitigation.

Reply:
(should have read ) needs to be updated to list the **removal** + mitigation.